deployer/templates/hawkular-metrics.yaml

```diff
@@ -97,9 +102,7 @@ objects:
           - "--hmw.truststore=/secrets/hawkular-metrics.truststore"
           - "--hmw.keystore_password_file=/secrets/hawkular-metrics.keystore.password"
           - "--hmw.truststore_password_file=/secrets/hawkular-metrics.truststore.password"
-          - "--hmw.jgroups_keystore=/secrets/hawkular-metrics.jgroups.keystore"
-          - "--hmw.jgroups_keystore_password_file=/secrets/hawkular-metrics.jgroups.keystore.password"
-          - "--hmw.jgroups_alias_file=/secrets/hawkular-metrics.jgroups.alias"
+          - "-DJGROUPS_PASSWORD=${JGROUPS_PASSWORD}"
```

Comment:
Ansible changes required: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/templates/hawkular_metrics_rc.j2#L36

since the deployer  is deprecated post 1.4.